pkg/binding/controller.go

```diff
@@ -179,9 +198,8 @@ func computeQPSFromNumGVRs(nGVRs int) float32 {
 		return rest.DefaultQPS
 	}
 	// in case too large, look at some value for reference
-	// https://github.com/kubernetes/kubernetes/pull/105520/files
-	if qps > 50.0 {
-		return 50.0
+	if qps > referenceQPSUpperBound {
```

Comment:
Why ?  Don't we allow overriding with explicit larger number ?

Reply:
This PR is not changing the fact that this is not configured.
_I_ have not asked for making it configurable because I have not seen a need for adding that complexity.

Reply:
I think the QPS should be upper-bounded somehow, so I looked around and found this reference value. Also, since we know that the `disposableClient` is going to do the full discovery, I just explicitly set the QPS to this upper bound for `disposableClient`.

Reply:
I think it's a good idea to be able to override through external parameter. Even for just some special test cases where we want to remove the limit completely. 
But I don't think my comment should block this PR.  
Up to you @waltforme if you want to change this in future PR.